src/main/java/io/cdap/e2e/pages/actions/CdfStudioActions.java

```diff
@@ -498,11 +505,40 @@ public static void closeStatusBanner() {
 
   /**
    * Click on the element and move.
+   *
    * @param pluginName plugin title.
-   * @param xOffset horizontal move offset.
-   * @param yOffset vertical move offset.
+   * @param xOffset    horizontal move offset.
+   * @param yOffset    vertical move offset.
    */
   public static void movePlugin(String pluginName, int xOffset, int yOffset) {
     ElementHelper.dragAndDropByOffset(CdfStudioLocators.locatePluginNodeInCanvas(pluginName), xOffset, yOffset);
   }
-}
+
+  /**
+   * Click on the Upload Button
+   */
+  public static void clickOnAddEntityButton() {
+    ElementHelper.clickOnElement(CdfStudioLocators.uploadFileButton);
+  }
+
+  public static void clickOnFixAllButton() {
+    ElementHelper.clickIfDisplayed(CdfStudioLocators.locatorOfFixAll());
+  }
+
+  public static void importPipeline(String filePath) throws URISyntaxException {
+    FileImportUtil.uploadFile(filePath);
+  }
+
+  public static void renameThePipeline() {
+    WaitHelper.waitForElementToBeOptionallyDisplayed(CdfStudioLocators.renamePipeline(), 100);
+    ElementHelper.clickOnElement(CdfStudioLocators.appendPipeline);
+    pipelineName = "TestPipeline-" + RandomStringUtils.randomAlphanumeric(10);
+    renamePipelineNameAndSave(pipelineName);
+  }
+  public static void renamePipelineNameAndSave(String pipelineName) {
```

Comment:
Remove this method. And reuse: fillPipelineNameAndSave to rename the pipeline

Reply:
Done.